File: server_nodejs/src/models/client_address.js
// Client address (UID, clientID, address, city, province, phone_number)
module.exports = (sequelize, Sequelize) => {
    class ClientAddress extends Sequelize.Model {}

    ClientAddress.init(
        {
            uid: {
                type: Sequelize.UUID,
                primaryKey: true,
            },
            clientid: {
                type: Sequelize.UUID,
                allowNull: false,
            },
            address: {
                type: Sequelize.STRING,
                allowNull: false,
            },
            city: {
                type: Sequelize.STRING,
                allowNull: false,
            },
            province: {
                type: Sequelize.STRING,
                allowNull: false,
            },
            phone_number: {
                type: Sequelize.STRING,
                allowNull: false,
            },
        },
        {
            sequelize,
            modelName: 'client_address',
            timestamps: true,
            paranoid: true,
            underscored: true,
        },
    );

    ClientAddress.associate = (models) => {
        ClientAddress.belongsTo(models.client_account, {
            foreignKey: 'clientid',
        });
        ClientAddress.hasMany(models.order, {
            foreignKey: 'shipping_addressid',
            as: 'order',
        });
    };

    return ClientAddress;
};

File: server_asp/src/models/ClientAddress.cs
using System;
using System.Collections.Generic;

namespace shieldtify.models;

public partial class ClientAddress
{
    public Guid Uid { get; set; }

    public Guid Clientid { get; set; }

    public string Address { get; set; } = null!;

    public string City { get; set; } = null!;

    public string Province { get; set; } = null!;

    public string PhoneNumber { get; set; } = null!;

    public DateTime CreatedAt { get; set; }

    public DateTime UpdatedAt { get; set; }

    public DateTime? DeletedAt { get; set; }

    public virtual ClientAccount Client { get; set; } = null!;

    public virtual ICollection<Order> Orders { get; set; } = new List<Order>();
}
